README.md

```diff
@@ -74,7 +74,7 @@ return [
 ### Step 3: Import routes
 
 ```yaml
-# config/routes/kms_froala_editor.yaml
+# config\routes.yaml 
```

Comment:
Not sure why the backslash instead of slash?
The docs are following the recipe here.

Reply:
@jmsche will update the backslash.